src/parser/warrior/arms/modules/talents/DefensiveStance.js

```diff
@@ -98,6 +104,21 @@ class DefensiveStance extends Analyzer {
       />
     );
   }
+  
+  suggestions(when) {
+    when(this.totalDamageLost).isGreaterThan(this.totalDamageMitigated)
+      .addSuggestion((suggest, dl, dr) => {
+        return suggest('The damage you inflicted were more reduced than those you received. Try to reduce the uptime of Defensive Stance when you take less damage.')
```

Comment:
Still not sure what this means ...

Are you trying to say they had more damage taken than damage dealt? As in the boss did more damage to them than they did to the boss? And that they should save Defensive Stance for when they are about to take damage?

Or are you trying to say they blocked more damage than they took? As in Defensive stance prevented more damage than damage that actually hit them?

I am also not quite sure what Damage Lost is

Reply:
So the totalDamageLost stand for the 10% decreased damage dealt by the player along the fight and the totalDamageMitigated for the 20% decreased damage taken by the player while he had the Defensive Stance active. As Defensive Stance could be activated and desactivated (after the 6sc cooldown) this could means that the Defensive Stance was active for too long during the fight

Reply:
oh does Defensive Stance reduce the damage you deal?

Reply:
Yes it's reduces the damage you take by 20% but you lost 10% of your damages while it's active :)

Reply:
Oh, i didnt realize Defensive Stance reduced the damage you deal also. 

Maybe something like this would be better then?
"While Defensive Stance was up, your damage done was reduced by more than the damage you mitigated. Ensure that you are only using Defensive Stance when you are about to take a lot of damage and that you cancel it quickly to minimize the time spent dealing less damage."

or some variation of that if its not entirely accurate.

Reply:
No problem at all, just want to make sure that people coming to the site understand it :)